open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-sync-connectors/ranger-connector/pom.xml

```diff
@@ -12,6 +12,11 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>ranger-connector</artifactId>
+
+    <properties>
+        <commons-collections4.version>4.0</commons-collections4.version>
```

Comment:
Since #22 has been merged you no longer need to add this property or include the version HERE.

Instead in the TOP level pom
 a) Add a property (as you have here) 
 b) Add a dependency in the DependencyManagement section of the top level pom along the lines of
```
           <dependency>
                <groupId>commons-collections</groupId>
                <artifactId>commons-collections</artifactId>
                <scope>compile</scope>
                <version>${commons-collections.version}</version>
            </dependency>
```
This is the existing entry. I see the artifact ids don't clash, so you can use yours alongside.

Reply:
I've done an update at top level pom - in egeria. I have updated the version of commons-collections and I changed some classes according to this new version.